src/Kinto.elm

```diff
@@ -472,6 +508,12 @@ extractKintoError statusCode statusMsg body =
                 |> ServerError statusCode statusMsg
 
 
+withQueryParam : ( String, String ) -> Request a -> Request a
+withQueryParam params builder =
+    -- XXX: Change the builder.url with the new param addition
```

Comment:
# Two options 
## Option 1

create our own Request that inhereit from RequestBuilder and add `queryParams` info so that we can update the url in `send`

## Option 2

Each time we used withQueryParams, we actually update the URL (extract the existing query params, concat the new ones and then update the builder.url value

@n1k0 @magopian what do you think?